app/config.py

```diff
@@ -58,6 +59,7 @@ class ApiProductionConfig(ApiConfig):
     DATABASE_PASSWORD = os.getenv('DATABASE_PASSWORD')
     DATABASE_HOST_ADDRESS = os.getenv('DATABASE_HOST_ADDRESS')
     DATABASE_NAME = 'whiteclaw'
+    WRITE_TO_AIRTABLE = True
```

Comment:
I think it would be better to encode this as an environment variable in addition to tying it to the prod config. Reason being that it might be easy for devs to make the mistake of setting FLASK_ENV=prod.

```suggestion
    WRITE_TO_AIRTABLE = os.getenv('WRITE_TO_AIRTABLE', False)
```